patterns/1-initial/innersource-hackathon.md

```diff
@@ -0,0 +1,80 @@
+## Title
+
+InnerSource Hackathon
+
+## Patlet
+
+In a company, initially only InnerSource enthusiasts are interested and practicing InnerSource during the early stages of InnerSource adoption; not all engineering teams are willing or have enough time and resources to adopt InnerSource. In this scenario, it is good to provide a safe space to try and adopt InnerSource through an InnerSource Hackathon event within the company.
+
+## Problem
+
+The company wants to adopt InnerSource as software development methodology but only those familiar with open source principles or those who understand the benefits of InnerSource, adopt it. It results in just a handful of InnerSource projects and is difficult to scale beyond that.
+
+## Context
+
+Scenario 1:
+The senior leadership believes in InnerSource and wants to drive it throughout the company. The engineers who are familiar with open source principles and/or understand the benefits of InnerSource are the early adopters. There is success with these initial pilot project and teams. Now the next step is to drive it across the company. There might be reluctance from engineering teams due to various factors like:
+
+* not familiar with InnerSource and being ignorant to know about it
+* not enough time to prioritize InnerSource, given the regular work deliverables
+* relunctance to changing ways of working when everything works well already
+* perception that InnerSource requires more work and responsibilities
+
+Scenario 2:
+Teams slowly start adopting InnerSource and open up their repositories. Some new projects are started as InnerSource projects from the start (that is open for contributions). However, there are not many contributors. It is also challenging to build a community around those projects, in spite of publishing the projects in an InnerSource Portal.  This could be due to various reasons like:
+
+* engineers do not have time to explore new InnerSource projects and contribute, outside regular work deliverables
+* no additional incentive to this effort apart from being acknowledged
+* lack of motivation by middle management
+
+## Forces
+
+Apart from the reasons listed in the Context section, there are other factors from different entities that prevent teams from adopting InnerSource like:
+
+* From middle management: perception of no direct benefit to the team apart from team upskilling and individual growth; eventually resulting in no motivation by the middle management to the engineers in trying out InnerSource ways of working.
```

Comment:
@spier we already have some patterns addressing push back like that - do you think it would make sense to link to those from here?

E.g.: [Change middle management mindset](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/change-the-middle-management-mindset.md), [Developer incentive alignment](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/developer-incentive-alignment-for-innersource-contribution.md), [Incentivise voluntary contributions](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/incentive-mechanisms-for-voluntary-contribution.md), [Overcome time constraints](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/overcoming-project-management-time-pressures.md)?

Reply:
Nice thought! We certainly could link to some of these, yes.

Just note that those other patterns have not reached the "Structured" [maturity level](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/meta/contributor-handbook.md#in-a-nutshell) yet. i.e. they may have not bee confirmed by an organization that used these patterns.

Reply:
Oh, right. Is there another good place to track these links?

Reply:
Have no good yet, no. 

Will just leave this conversation open, while already going ahead and merging the PR.